front/app/containers/GenericStudySectionPage/GenericStudySectionPage.tsx

```diff
@@ -45,19 +45,26 @@ class QueryComponent extends Query<
 class GenericStudySectionPage extends React.PureComponent<
   GenericStudySectionPageProps
 > {
-  renderItem = (key: string, value: string | null) => {
+  renderItem = (key: string, value: string | number | null) => {
     const name = pipe(
       snakeCase,
       split('_'),
       map(capitalize),
       join(' '),
     )(key);
+
+    const text = pipe(
+      value => value.toString(),
+      split(/\n{2,}/),
+      arr => arr.map((paragraph, i) => <p key={i} style={{ margin: ".5em 0" }}>{paragraph}</p>)
```

Comment:
what is this line doing and do we want this margin/style hard coded here or handles some other way?

Reply:
This line takes each line (which is determined by being preceded two or more line breaks) and wraps the text of the line in a paragraph element. Ideally the style wouldn't be hard coded, but there are already hard coded styles in this file, and this specifically relates to the table, like the other styles do